roles/kubernetes/loadbalancer/metallb/defaults/main.yaml

```diff
@@ -7,8 +7,8 @@ METALLB_NAME: metallb
 METALLB_NAMESPACE: kube-system
 METALLB_CHART_NAME: metallb
 METALLB_CHART_URL: https://metallb.github.io/metallb
-METALLB_CHART_VERSION: 0.13.7
-METALLB_KUBE_CONFIG: ~/.kube/config
+METALLB_CHART_VERSION: 0.14.3
+METALLB_KUBE_CONFIG: "{{ ansible_env.HOME }}/.kube/config"
```

Comment:
Note for reviewers:
This technically resolves to the same value, I had to implement this change due to a bug in the `kubernetes.core` ansible module that been around for a while. Where if you specify the kubeconfig explicitly as `~/.kube/config` it will cause the module to fail. Users who use this default value will see no changes in behavior.
 
There is a PR that fixes this bug on the module side, but it has been inactive for a while
 https://github.com/ansible-collections/kubernetes.core/pull/654